internal/cli/atlas/kubernetes/operator/install.go

```diff
@@ -170,7 +174,13 @@ The key is scoped to the project when you specify the --projectName option and t
   atlas kubernetes operator install --targetNamespace=<namespace> --orgID <orgID> --import
 
   # Install and import objects from a specific project:
-  atlas kubernetes operator install --targetNamespace=<namespace> --orgID <orgID> --projectName <project> --import`,
+  atlas kubernetes operator install --targetNamespace=<namespace> --orgID <orgID> --projectName <project> --import
+
+	# Install the operator and disable deletion protection:
+	atlas kubernetes operator install --resourceDeletionProtection=false
```

Comment:
is this flag supposed to be a parallel with the cluster termination protection feature?

the way we do the boolean flags is slightly different, instead of assigning false or true to the flag, we can make it a boolean. here's the atlas clusters flag example:    
```
 --enableTerminationProtection   Enables termination protection for your cluster. You can't delete a cluster with termination protection enabled.
```

if you want to have it on by default, you could have it called "--disableDeletionProtection"

Reply:
No, these flags are not designed to work with the `enableTerminationProtection`. Also, we'd like to make them explicit (as mentioned in the scope), so that users understand they explicitly disable the operator "deletion protection" logic.